### PR TITLE
refactor(plugin-meetings): use deviceCapabilities instead of individual flags

### PR DESCRIPTION
**Pull request by sundayu**
# COMPLETES [SPARK-443854](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-443854)

## This pull request addresses

add capabilities for simultaneous interpretation.

## by making the following changes

deviceCapabilities add HOST_CONTROL_SI_SUPPORTED and INTERPRETER_CONTROL_SI_SUPPORTED

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by mickelr — APPROVED** [review]
LGTM

File: packages/@webex/plugin-meetings/src/meeting/request.ts [hunk 1]
```diff
@@ -177,6 +182,14 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       deviceCapabilities.push(ANNOTATION.ANNOTATION_ON_SHARE_SUPPORTED);
     }
 
+    if (siHostSupported) {
```
Comment (Coread) [inline packages/@webex/plugin-meetings/src/meeting/request.ts:185]: question: This is fine, but I thought the `deviceCapabilities` argument was added so we no longer needed to add individual flags like this?

Comment (sundayu) [reply packages/@webex/plugin-meetings/src/meeting/request.ts:185]: Thanks @Coread for pointing out the question.
It does duplicate and I have the same confusion. There are two ways to pass device capabilities in the code. I was referencing breakouts and annotation.
If `deviceCapabilities` argument is sufficient, are we considering unifying into one?

Comment (Coread) [reply packages/@webex/plugin-meetings/src/meeting/request.ts:185]: yes please go ahead and unify into one

**Comment by sreenara** [comment]
This has been in draft for over a year. Therefore, closing it.

**Pull request closed without merging**